tests/test_admin.py

```diff
@@ -63,6 +63,9 @@ def test_add_user(self):
              'notes':['notes'],
              'username':['foo'],
              'status':['active'],
+             'email':['foobar'],
```

Comment:
_Extremely_ small nitpick- could you change this to be something like foobar@baz.com to make this parameter's format more explicit?